3rdparty/libconfig/grammar.c

```diff
@@ -95,8 +97,8 @@
 #include "scanctx.h"
 
 /* these delcarations are provided to suppress compiler warnings */
-extern int libconfig_yylex();
-extern int libconfig_yyget_lineno();
+extern int libconfig_yylex(YYSTYPE *,yyscan_t);
+extern int libconfig_yyget_lineno(yyscan_t);
```

Comment:
```suggestion
```
Since you now correctly include scanner.h, the definitions in there will be used and the definitions in there are already correct.
So all this can safely be removed.